Makefile

```diff
@@ -174,14 +174,22 @@ check-plugin:
 	@echo "checking plugin"
 	cd ./plugin/scheduler_example && $(MAKE) evictLeaderPlugin.so && rm evictLeaderPlugin.so
 
+GOLANGCILINT := $(shell command -v golangci-lint 2>/dev/null)
 static: export GO111MODULE=on
 static:
```

Comment:
```suggestion
static: install-go-tools
```

Why not just call `install-go-tools` here rather than throwing a message?

Reply:
Good idea, fixed